repository/src/main/java/org/apache/atlas/repository/store/graph/v2/AtlasTypeDefGraphStoreV2.java

```diff
@@ -540,7 +540,7 @@ TypeCategory getTypeCategory(AtlasBaseTypeDef typeDef) {
      * update the given vertex property, if the new value is not-blank
      */
     private void updateVertexProperty(AtlasVertex vertex, String propertyName, String newValue) {
-        if (StringUtils.isNotBlank(newValue)) {
+        if (newValue != null) {
```

Comment:
This is a generic change, let us test for a few mandatory attributes as well e.g "name"

Reply:
Updating property will be skipped in case newValue == null, how does it clear the attribute value

Reply:
Make sense , added new method `updateVertexPropertyEvenIfValueIsNull` which updates description value even if it is null.